charms/xenial/sojobo-api/config.yaml

```diff
@@ -28,7 +29,8 @@
       \ key is used that will already be installed on the machine, and for PPA sources\
       \ where the package signing key is securely retrieved from Launchpad.\n"
     "type": "string"
-    "default": ""
+    "default": |
+       - "C8068B11"
```

Comment:
Select the "Technical details about this PPA" dropdown here -> https://launchpad.net/~juju/+archive/ubuntu/stable

Also line 8 here -> http://paste.ubuntu.com/23454428/